pkg/controlsvc/reload.go

```diff
@@ -87,10 +95,20 @@ func parseConfigForReload(filename string, checkReload bool) error {
 			return err
 		}
 		cfg := string(cfgBytes)
+		fmt.Println(cfg)
+
+		if strings.Contains((cfg), "log-level") {
+			args := strings.Split(cfg, ":")
+			if len(args) > 1 {
```

Comment:
this check has been duplicated within the `setLogLevelFromConfig` function, any reason for this?